sample/src/main/kotlin/io/getstream/chat/sample/feature/channels/ChannelsFragment.kt

```diff
@@ -35,21 +34,30 @@ class ChannelsFragment : Fragment(R.layout.fragment_channels) {
 
         Chat.getInstance().setUser(user, loggedInUser.token, object : InitConnectionListener() {
             override fun onSuccess(data: ConnectionData) {
-                ChatLogger.instance.logD("ChannelsList", "User set successfully")
+                Timber.d("User set successfully")
             }
             override fun onError(error: ChatError) {
-                Timber.e(error)
+                Timber.e("Failed to set user")
             }
         })
+
         ChannelsViewModelImpl().apply {
             bindView(channelsList, this@ChannelsFragment)
             state.observe(viewLifecycleOwner, Observer {
                 when (it) {
                     is ChannelsViewModel.State.Loading -> {
                         loadingProgressbar.visible(it.isLoading)
+                        emptyStateLabel.visible(false)
```

Comment:
Would be great if we have a customizable emptyView into the `ChannelListView`, then all this logic could be handled into the view, and the devs only need to design that view

Reply:
Sounds good